rules/record-better-audio/rule.md

```diff
@@ -14,81 +14,89 @@ created: 2020-11-27T22:06:55.000Z
 archivedreason: null
 guid: d8e3cd29-5d87-442e-998b-25d5649efbc3
 ---
-
 Audio is one of the most important things to get right in a video as many viewers will tune out if the sound is not good, because if you can't hear someone properly it makes it more difficult to understand them.
 
 <!--endintro-->
+
 ### Background Noise
+
 Having sounds in the background during a recording can be very distracting, and sometimes irritating to a viewer, e.g. construction, traffic, someone in the kitchen banging pots and pans.
 
 Try these tips to minimize unwanted noise:
-1.	Timing – schedule your recording during a quiet part of the day (e.g. avoid meal-times and breaks)
-2.	Block out & dampen sounds – reduce the amount of noise by physically obstructing it:
-    -	Close all doors and windows - a sealed space will reduce a large amount of noise
-    -	Create a barrier – put obstacles in between your microphone and the unwanted noise (e.g. use a wall divider, or stack some boxes to make a wall)
-    -	Use Carpet/Rugs – soft materials will cut down on echo and ambient noise
-3.	Mic distance – putting your microphone closer to your audio source will make your desired audio louder, and unwanted noise will be less noticeable
+
+1. Timing – schedule your recording during a quiet part of the day (e.g. avoid meal-times and breaks)
+2. Block out & dampen sounds – reduce the amount of noise by physically obstructing it:
+
+   * Close all doors and windows - a sealed space will reduce a large amount of noise
+   * Create a barrier – put obstacles in between your microphone and the unwanted noise (e.g. use a wall divider, or stack some boxes to make a wall)
+   * Use Carpet/Rugs – soft materials will cut down on echo and ambient noise
+3. Mic distance – putting your microphone closer to your audio source will make your desired audio louder, and unwanted noise will be less noticeable
 
 ### Silent Spaces – Record “Room Tone”
+
 Room Tone is the ambient noise in a room when no sound is being produced, like a room’s audio fingerprint.
 The sound is shaped by many things, including the room’s size, shape, and objects, as well as noise created from appliances, e.g. refrigerators.
 
 As these sounds are constant and consistent, editing software that can remove a lot of this noise, making your audio recordings much nicer.
```

Comment:
Should this line be "As these sounds are constant and consistent, editing software that can remove a lot of this noise makes your audio recordings much nicer."
- I know you didn't make changes to this line but let's fix it anyway :)